cmd/lachesis/commands/run_lachesis.go

```diff
@@ -1,8 +1,10 @@
+package commands
```

Comment:
Build tags (comment lines starting with // +build ) must appear near the top of the file, preceded only by blank lines and other line comments. (see https://golang.org/pkg/go/build/ ).
Thus  package command must be put back down under build tag.

Reply:
Updated, thanks for catching!